protocol-tests/src/main/java/fixtures/bodycomplex/ArrayAsyncClient.java

```diff
@@ -167,4 +168,106 @@ public Mono<Response<Void>> putEmptyWithResponse(BinaryData complexBody, Request
     public Mono<Response<BinaryData>> getNotProvidedWithResponse(RequestOptions requestOptions) {
         return this.serviceClient.getNotProvidedWithResponseAsync(requestOptions);
     }
+
+    /**
+     * Get complex types with array property.
+     *
+     * @throws com.azure.core.exception.HttpResponseException thrown if the request is rejected by server.
+     * @throws ClientAuthenticationException thrown if the request is rejected by server on status code 401.
+     * @throws ResourceNotFoundException thrown if the request is rejected by server on status code 404.
+     * @throws ResourceModifiedException thrown if the request is rejected by server on status code 409.
+     * @throws RuntimeException all other wrapped checked exceptions if the request fails to be sent.
+     * @return complex types with array property on successful completion of {@link Mono}.
+     */
+    @Generated
+    @ServiceMethod(returns = ReturnType.SINGLE)
+    public Mono<ArrayWrapper> getValid() {
```

Comment:
model should not be in DPG.

I don't know what's wrong. It is overridden (temporarily) by GENERATE_CONVENIENCE_METHODS env var in ClientModelUtil.

Reply:
OK, I am removing the usage of dev environment variable in PR https://github.com/Azure/autorest.java/pull/1835